staging/src/k8s.io/kubectl/pkg/cmd/auth/auth.go

```diff
@@ -19,10 +19,14 @@ package auth
 import (
 	"github.com/spf13/cobra"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/cli-runtime/pkg/printers"
+	"k8s.io/kubectl/pkg/util/term"
 
 	cmdutil "k8s.io/kubectl/pkg/cmd/util"
 )
 
+var warningPrinter *printers.WarningPrinter
```

Comment:
We'll want all of these to be part of the command options where they are used and not package scoped.

Reply:
fixed